plugins/deployment_manager/DeploymentManager.ts

```diff
@@ -310,6 +310,13 @@ export class DeploymentManager {
     return this.contractsCache;
   }
 
+  /* Gets all the contracts, connected to signer, as an object */
+  async getContracts(signer?: SignerWithAddress): Promise<{ [alias: Alias]: Contract }> {
```

Comment:
following on from: https://github.com/compound-finance/comet/pull/510#discussion_r943935518

whatever the solution, it seems like we should avoid having two functions called `contracts` and `getContracts` that do very similar, but importantly-different things

Reply:
You should just use `getContracts`, and we'll merge them soon, this will be an easy fix just takes a little time that we don't have right now

Reply:
`getContracts` is really just more convenient though so scripts can do e.g. `const { configurator, comet } = await deploymentManager.getContracts()` I don't think there's any trap here